fabtests/common/hmem.c

```diff
@@ -126,11 +138,22 @@ int ft_hmem_alloc(enum fi_hmem_iface iface, uint64_t device, void **buf,
 	return hmem_ops[iface].alloc(device, buf, size);
 }
 
+int ft_hmem_alloc_host(enum fi_hmem_iface iface, uint64_t device, void **buf,
```

Comment:
I understand the convenience of reusing ft_host_alloc() for the alloc_host op, but it's weird to have the 'device' parameter for a host memory allocation API.

Reply:
Yes. I thought about it.

Though the parameter `device` is  not used by cuda. But, maybe some future device will need to allocate host buffer just for certain device?

Reply:
My question is why `device` is an argument in `ft_host_alloc`. It seems it's not used.

Reply:
Similar question for other `ft_host*` API like `ft_host_memset`

Reply:
Never mind, I think the reason is `FT_HMEM_SYSTEM` is also an interface and they share the same API parameters.

Reply:
None of these calls require a stable API.  Implement what's needed and makes sense.  We can't guess what some future theoretical device API may have as a requirement, so it doesn't make sense to complicate the API we need in anticipation of what might be needed.

Reply:
Make sense. I removed the `device` parameter in and introduced `ft_default_alloc_host` and `ft_default_free_host` in new revision.